sre-recipes/sandboxctl

```diff
@@ -92,6 +116,9 @@ def main(action, recipe_name):
     elif action == 'verify':
         logging.info('Verifying %s', recipe_name)
         recipe.verify()
+    elif action == 'loadgen-step':
+        logging.info('Restarting Load Generator in Step Pattern')
+        restart_loadgen(pattern=LoadPattern.STEP)
```

Comment:
When I tried running `sandboxctl loadgen-step`, it said I was missing the recipe argument